src/olympia/addons/admin.py

```diff
@@ -167,6 +168,32 @@ def total_ratings_link(self, obj):
             count=obj.total_ratings)
     total_ratings_link.short_description = _(u'Ratings')
 
+    def reviewer_links(self, obj):
```

Comment:
We haven't looked at optimizing `AddonAdmin` at all yet, I don't think there is even an `assertNumQueries` test for the `changelist` view, and this particular case is tough to optimize, but it's going to add one to two (depending on whether the add-ons have a `current_version` set or not) queries _for each add-on_ in the list :( This might make it a lot slower, even if the queries should be fast...

Reply:
ok, I filed https://github.com/mozilla/addons-server/issues/13862 to add a test case at least. Depending on the outcome, we might want to spend some time optimizing the `changelist` view.